---
layout: default
---

I'm a broadly trained evolutionary biologist and ecologist interested in the processes that generate and maintain biological diversity. My research merges natural history and field biology with genomics and population genetics theory, typically working at the intersection of ecological and evolutionary time scales. I collect empirical data from wild populations of birds and insects and validate models using evolutionary simulations and theory. I'm especially passionate about tropical mountains, natural history museums, and trying to understand how your tools work.  

![Areas of research focus.](/images/research_statement.png)

## Environmental drivers of speciation and hybridization  

After decades of imprecise verbal models and a general sense that Darwin [misnamed his famous book](https://doi.org/10.1098/rstb.2008.0081),
a resurgence of theory and the onset of high throughput DNA sequencing has reinvigorated the study of speciation. We're now faced with reconciling
biological intuition and evidence from species distributions (that suggest geographic isolation is crucial to most speciation events) with new theory
and genomic evidence (which suggest that gene flow and selection are rampant throughout the evolutionary history of diverging lineages). What barriers, then, commonly involved in maintaining species limits in the face of introgressive hybridization? In contrast to genetic incompatibilities and fixed differences in phenotype, habitat isolation has received relatively little attention. My National Science Foundation-funded [dissertation research](https://doi.org/10.1101/589044) demonstrated selection across elevation was sufficient to counteract a long history of gene flow in *Syma* kingfishers in New Guinea, and developed theory for the [role of glacial cycles in mediating speciation rates](https://doi.org/10.1101/758664). I'm currently working with [Scott Taylor and Georgy Semenov at CU Boulder](https://www.colorado.edu/lab/taylor/) to model the demographic history of hybridizing wagtails, and developing a new system that uses anthropogenic disturbance-induced hybridization to understand the strength of habitat isolation in tropical mountains.   

![](/images/manhattan_new.png)
*The genomic landscape of divergence in* Syma *kingfishers*

## Species ranges and population differentiation  

What determines whether a species expands its range, and whether it differentiates across it? The answer is partly related to physical geography and dispersal ability. To describe these factors, I have previously used both DNA sequence data and natural history museum collection metadeta, identifying [a migratory divide](https://doi.org/10.1086/695439) and [circular migration route](https://doi.org/10.7717/peerj.1871) in painted buntings, as well as unexpected population genetic structure in [highly vagile island white-eyes](https://doi.org/10.1016/j.ympev.2015.08.018). But it is also related to the interaction between gene flow and local adaptation, and its consequences for demography. My [previous postdoctoral research](https://doi.org/10.1101/783233) with Dr. Kimberly Sheldon asked whether narrow thermal niches are sufficient to generate isolation-by-environment at small spatial scales, and what the consequences of high gene flow are for the evolution of elevational ranges (*Ecology and Evolution*, in press). My current work as a NSF postdoctoral fellow at the [University of New Mexico](https://www.unmornithology.org/) combines many of these interests, investigating the population genetics of species range limits in Andean birds.  

![](/images/range_limits.png)
*The effect of maladaptive alleles on effective population size at a range limit*  

## How do we know what we know?  

Biology is messy. Massive DNA sequence datasets require computational analyses and interpretation at the limits of our current knowledge, adding another layer of noise to the signals of biological processes. Because it's useful to understand what can be contributing to uncertainty in your conclusions, I'm interested in methods for generating and analyzing genomic data. I've worked to validate laboratory [methods for collecting population genomic data from degraded DNA sources](https://doi.org/10.1002/ece3.3065),
and to understand how coalescent theory can explain why bioinformatic filtering steps can [alter inferred patterns of population genetic structure](https://doi.org/10.1111/1755-0998.12995). I have also used a classic taxonomic conundrum in ornithology to demonstrate [why incomplete geographic sampling can mislead us](https://doi.org/10.1093/sysbio/syz027) when we try to understand phylogenetic relationships and species limits.

![](/images/sfs_ridge.png)
*Mutational age and minor allele frequencies across a coalescent tree*

## Tropical natural history  

Natural history provides the raw material for biological hypotheses. The overwhelming grandeur of tropical montane biodiversity has played a significant role in determining my interests as a biologist, but for many species (the vast majority?) we lack detailed information on distributions, abundances, life history, behavior, reproduction, and other basic biological parameters. I am committed to the slow but invaluable work of contributing these fundamental data, and have written notes on [high alpine bird communities](https://www.researchgate.net/publication/309534829_Notes_on_birds_and_logistics_from_the_south_side_of_Mount_Giluwe_SHP), nesting biology in [a widespread swallow](http://journals.sfu.ca/ornneo/index.php/ornneo/article/view/402), and the surviving bird communities of the Mariana Islands (In press, *Wilson Journal of Ornithology*). I am particularly interested in natural history of the rainforests of New Guinea and Solomon Islands, where I have spent significant time and
[reported on community conservation initiatives as a photojournalist](https://www.biographic.com/posts/sto/where-the-rainforest-meets-the-road).  

![](/images/brushturkey.jpg)
*A high elevation Wattled Brushturkey (megapode) egg from 1800m on Mt. Wilhelm, Papua New Guinea*

[back](./)
